web/views/page_view.ex

```diff
@@ -15,6 +15,19 @@ defmodule Tudo.PageView do
       |> List.to_tuple
       |> get_label_font_color
     end)
+    |> filter_rendered_labels
+  end
+
+  def allowed_labels, do: ["help wanted", "awaiting-review", "discuss",
```

Comment:
could this be a `defp`?

Reply:
Probably for the moment yeah... We were thinking we'd add a _filter by label_ feature using `allowed_labels` as options, so we preemptively made it public.

Reply:
if thats next on the backlog then id say lets leave it how it is, if not ld suggest changing it due to law of "saying we will tidy up later depending on something else but naturally forget" which I certainly fall victim to :)

Reply:
That's what we're going to work on now so will merge this.